src/helpers.ts

```diff
@@ -62,22 +63,19 @@ function hasNodeExecutable(rootPath: string, executable: string): boolean {
  */
 export function pathToJest(pluginSettings: IPluginSettings) {
   if (pluginSettings.pathToJest) {
-    if (isBootstrappedWithCRA(pluginSettings.rootPath)) {
-      return 'npm test --'
-    }
     return normalize(pluginSettings.pathToJest)
   }
 
-  const platform = process.platform
-  if (platform === 'win32' && existsSync(join(pluginSettings.rootPath, 'node_modules', '.bin', 'jest.cmd'))) {
-    return normalize(join(pluginSettings.rootPath, 'node_modules', '.bin', 'jest.cmd'))
-  } else if (
-    (platform === 'linux' || platform === 'darwin') &&
-    existsSync(join(pluginSettings.rootPath, 'node_modules', '.bin', 'jest'))
-  ) {
-    return normalize(join(pluginSettings.rootPath, 'node_modules', '.bin', 'jest'))
+  if (isBootstrappedWithCRA(pluginSettings.rootPath)) {
+    return 'npm test --'
+  }
+
+  const defaultPath = normalize('node_modules/.bin/jest')
+  if (existsSync(join(pluginSettings.rootPath, defaultPath))) {
+    return defaultPath
```

Comment:
This would remove the check for `jest.cmd` and fail on Windows systems.

Reply:
This wouldn't fail since on Windows both files, 'node_modules/.bin/jest' and 'node_modules/.bin/jest.cmd' are present. But if we already check for the existence of the file, we probably really should keep caring about the extension.